File: Chessboard Distance.c
#include<stdio.h>
#include<math.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x1,y1,x2,y2,a,b,max;
        scanf("%d%d%d%d",&x1,&y1,&x2,&y2);
        a=abs(x2-x1);
        b=abs(y2-y1);
        if(a>b)
        max=a;
        else
        max=b;
        printf("%d\n",max);
    }
}

File: Chessboard Distance.java
import java.util.*;
import java.lang.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int c=sc.nextInt();
            int d=sc.nextInt();
            int s1=Math.abs(a-c);
            int s2=Math.abs(b-d);
            int x=Math.max(s1,s2);
            System.out.println(x);
        }
    }
}